src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectCapabilityResolver.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.Razor;
+
+internal abstract class ProjectCapabilityResolver
```

Comment:
Can you make this an interface, `IProjectCapabilityResolver`

Reply:
Done